dbt_style_guide.md

```diff
@@ -1,237 +1,510 @@
 # dbt Style Guide
 
+## Model Organization  
+Our models (typically) fit into two main categories:
+
+| Category | Description                                             |
+|----------|---------------------------------------------------------|
+| Staging  | Contains models which clean and standardize data        |
+| Marts    | Contains models which combine or heavily transform data |
+  
+Things to note:
+- There are different types of models
+that typically exist in each of the above categories.  
+See [dbt Conventions](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md#dbt-conventions) 
+for more information. 
+
+- Read [How we structure our dbt projects](https://docs.getdbt.com/guides/best-practices/how-we-structure/1-guide-overview) to see how we typically structure our projects and further thoughts around organization.
+
+## Modeling Conventions
+- Only models in `staging` should select from [sources](https://docs.getdbt.com/docs/building-a-dbt-project/using-sources)
+- Models not within the `staging` folder should select from [refs](https://docs.getdbt.com/reference/dbt-jinja-functions/ref).
+- The following are the DAG stages that we tend to utilize:
+  <details>
+
+  <summary>Common</summary>
+
+    | dag_stage | Typically found in | description                                                        |
+    |-----------|--------------------|--------------------------------------------------------------------|
+    | seed_     | /data              | <li> Indicates a data set created from `dbt seed`. |

```

Comment:
Debated commenting since this is minor, but since the newer projects only name this folder '/seeds', I think we should reference that here to not confuse new users. Either change '/data' to '/seeds' or say '/data or /seeds'

Reply:
Oof really great catch and *definitely* something we should change!

Reply:
[Done!](https://github.com/dbt-labs/corp/pull/63/commits/03d6571053c7fda6d73b0dc0cbcb733dd0faa863)